dist/.ddev/wunderio/core/__varnish.vcl

```diff
@@ -1,3 +1,4 @@
+# NOT USED, __ prefix means it's under evaluation if to be removed.
```

Comment:
I've originally copied these files from our Drupal template. __ files I've marked as yet to do or investigate if this is needed.